component-samples/component-http-client-to0-sample/pom.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+     Copyright 2020 Intel Corporation
+     SPDX-License-Identifier: Apache 2.0
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>component-samples</artifactId>
+    <groupId>org.fido.iot</groupId>
+    <version>0.5-SNAPSHOT</version>
+  </parent>
+
+  <name>Component HTTP TO0 Client Sample</name>
+  <artifactId>component-http-client-to0-sample</artifactId>
+  <packaging>jar</packaging>
+
+  <properties>
+    <jakson.dataformat.cbor.version>2.11.0</jakson.dataformat.cbor.version>
+    <commons-codec.version>1.14</commons-codec.version>
+    <tomcat.version>9.0.40</tomcat.version>
```

Comment:
tomcat.version => 9.0.44
bcpkix.version => 1.68 

These are as per new snyk findings (the PR to fix it at other places has been merged yesterday)